Modules/Chatroom/classes/class.ilChatroom.php

```diff
@@ -205,6 +207,7 @@ public static function byRoomId(int $room_id, bool $initObject = false) : ?ilCha
             $room = new self();
             $room->initialize($row);
 
+	    // CR: Incompatible types: Expected property of type '\ilObjChatroom|null', 'bool|\ilObject' provided  
```

Comment:
This is a `False Positive`, `ilObjectFactory::getInstanceByObjId` will return a PHP type depending on the id's object type.